luminance/src/shader/stage.rs

```diff
@@ -87,7 +87,7 @@ impl Stage {
 
         gl::DeleteShader(handle);
 
-        log.set_len(log_len as usize);
+        log.set_len(log_len as usize - 1);
```

Comment:
I really am not sure about that.

![Screenshot 2020-02-06 at 15 43 29](https://user-images.githubusercontent.com/506592/73947252-7416e180-48f7-11ea-94bc-77a6cabcafc8.png)

Reply:
Indeed, I’ll have a look at why I got a null terminator then, probably a Mesa bug. Thanks!